src/main/java/kr/dgucaps/caps/domain/wiki/repository/WikiRepository.java

```diff
@@ -24,6 +25,7 @@ public interface WikiRepository extends JpaRepository<Wiki, Long> {
 
     List<Wiki> findFirst10ByIsDeletedFalseOrderByCreatedAtDesc();
 
-    @Query("SELECT w.title FROM Wiki w WHERE w.isDeleted = false")
-    List<String> findAllTitle();
+    @Query("SELECT w.jamo FROM Wiki w WHERE w.isDeleted = false AND w.jamo LIKE %:input% ORDER BY w.jamo ASC LIMIT 7")
+    List<Wiki> findAutocompleteWiki(@Param("input") String input);
```

Comment:
이렇게만 해도 될 것 같아!!
```suggestion
    List<Wiki> findByJamoStartsWith(String jamo);
```

7개만 가져오려면
```suggestion
    List<Wiki> findFirst7ByJamoStartsWith(String jamo);
```
처럼 해도 될 것 같은데 이거는 더 정확한(유사한) 7개 나오는지 테스트 필요!!

Reply:
정렬 순서는 필요없나요?

Reply:
일단은 이대로 해보고 테스트 후에 수정하는 거로 해봅시다